presidio-image-anonymizer/presidio_image_anonymizer/entities/api_request_convertor.py

```diff
@@ -0,0 +1,44 @@
+import json
+import logging
+from typing import Tuple, Union, io
+
+from PIL import Image
+
+from presidio_image_anonymizer.entities import InvalidParamException
+
+logger = logging.getLogger("presidio-image-anonymizer")
+
+
+def get_json_data(data: str) -> dict:
+    try:
+        if not data:
+            return {}
+        return json.loads(data.replace("'", "\""))
+    except Exception as e:
+        logger.error(
+            f"failed to parse json from string '{data}' with error {e}")
+        raise InvalidParamException(f"Invalid json format \'{data}\'")
+
+
+def color_fill_string_to_value(json_params: dict) -> Union[
+    int, Tuple[int, int, int]]:
+    filling_str = json_params.get("color_fill")
+    try:
+        if not filling_str:
+            return 0, 0, 0
+        filling_str_split = filling_str.split(',')
+        if len(filling_str_split) == 1:
+            return int(filling_str_split[0])
+        return tuple(map(int, filling_str_split))
```

Comment:
This could return Tuple[int, ...] instead as the code does not guarantee 1 or 3 ints -- could be 2 or 4 for example

Reply:
I've added a check if != 3 we throw an exception.